src/components/Admin/Products/ProductTable.jsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import { GoPencil } from 'react-icons/go'
+import Loading from '../../Loading/Loading'
+import './ProductTable.scss'
+
+function ProductTable ({ products, isLoading, onOpenEdit }) {
+  const headboard = ['Name', 'Description', 'Code', 'Category', 'Price', 'Stock', '']
+  const isListEmpty = !products || products.length === 0
```

Comment:
`const isListEmpty = !products || !products.length`

Reply:
I will make the change although I see it with more meaning this way but I will make this change anyway :)